www/contribute/wanted-ebooks.php

```diff
@@ -694,17 +694,14 @@
 			<li>
 				<p><a href="https://www.gutenberg.org/ebooks/45502">How the Other Half Lives</a> by Jacob Riis</p>
 			</li>
-			<li>
-				<p><a href="https://www.gutenberg.org/ebooks/9296">Clarissa</a> by Samuel Richardson (<a href="https://github.com/drgrigg/samuel-richardson_clarissa-harlowe">partially completed</a>, available for finishing but will take a large amount of time)</p>
-			</li>
 			<li>
 				<p><a href="https://www.gutenberg.org/ebooks/7469">Daniel Deronda</a> by George Eliot (part of the BBC 2015 collection)</p>
 			</li>
 			<li>
 				<p><a href="https://www.gutenberg.org/ebooks/10800">The Anatomy of Melancholy</a> by Robert Burton</p>
 			</li>
 			<li>
-				<p><a href="https://www.gutenberg.org/ebooks/author/783">Short stories by P. G. Wodehouse (Note that we already have all Jeeves and School stories. Careful research is required to compile an omnibus of the rest.)</a></p>
+				<p><a href="https://www.gutenberg.org/ebooks/author/783">Short story collections by P. G. Wodehouse</a> (Note that we already have all Jeeves, School and Ukridge Stories completed. A spreadsheet is available list the rest.)</p>
```

Comment:
Typo in `is available list`, can you fix?

Reply:
Sorry.